e1000d/src/device.rs

```diff
@@ -6,6 +7,9 @@ use syscall::flag::O_NONBLOCK;
 use syscall::io::Dma;
 use syscall::scheme::Scheme;
 
+// TODO: One // line here to describe what this GROUP of constants is
+// TODO: one /// line describing each constant. Just defining the acronym
```

Comment:
Yep, that sounds good. The specifications I pulled the magic values from often have a description of each one.